frontend/Ingredishare/app/src/main/java/com/example/andriod/ingredishare/IngredientListActivity.java

```diff
@@ -7,13 +7,6 @@
 import android.location.Location;
 import android.location.LocationManager;
 import android.os.Bundle;
-import android.support.v4.app.ActivityCompat;
-import android.support.v4.content.ContextCompat;
-import android.support.v7.app.AlertDialog;
-import android.support.v7.app.AppCompatActivity;
-import android.support.v7.widget.LinearLayoutManager;
-import android.support.v7.widget.RecyclerView;
-import android.support.v7.widget.Toolbar;
```

Comment:
import statements just changed from `android.support.v#...` to `androidx.appcompat...`